.github/workflows/ci.yml

```diff
@@ -14,10 +14,10 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        otp_vsn: ['24', '25', '26']
+        otp_vsn: ['25', '26']
```

Comment:
Do you mind adding `'27'` here, @vkatsuba ?

Reply:
OK

Reply:
btw - it strange that in CI we got this issue... locally I use OTP 24 and it builded as well for me...

Reply:
You probably had a cached version of the library.

Reply:
Nope. Not that.
Because the libs all have version numbers… https://github.com/inaka/cowboy_swagger/blob/24334d88d76770f32ee0eed2218c94e7e2eb8720/rebar.config#L24-L29 
No idea what's going on.

Reply:
Try running `rebar3 upgrade --all`. Let's see what it tells you.

Reply:
CI updates as well https://github.com/vkatsuba/cowboy_swagger/actions/runs/11253721613

The rebar3 upgrade show all green

Reply:
Well... `rebar3 test` show me that error locally.